src/components/BulkEnrollmentPage/CourseSearchResults.test.jsx

```diff
@@ -136,6 +147,28 @@ describe('<CourseSearchResults />', () => {
     const wrapper = mount(<CourseSearchWrapper props={{ ...defaultProps, isSearchStalled: true }} />);
     expect(wrapper.find(Skeleton)).toHaveLength(1);
   });
+  it('shows selection options when at least one course is selected', () => {
+    renderWithRouter(<CourseSearchWrapper {...defaultProps} />);
+    const rowToSelect = screen.getByText(testCourseName2).closest('tr');
+    userEvent.click(within(rowToSelect).getByTestId('selectOne'));
```

Comment:
you could definitely just do:
```
userEvent.click(screen.getAllByTestId('selectOne')[0]);
```

I honestly just wanted an excuse to use `within()` since I just learned about it (but it's more useful where we actually have unique identifiers up one level, whereas here they both are under generic table row and the same rowGroup with no differentiating ids or elements)

Reply:
I love it.